packages/build/src/plugins_core/edge_functions/index.ts

```diff
@@ -66,19 +66,31 @@ const coreStep = async function ({
       systemLogger: featureFlags.edge_functions_system_logger ? systemLog : undefined,
       internalSrcFolder: internalSrcPath,
     })
+    const metrics = getMetrics(manifest)
 
     systemLog('Edge Functions manifest:', manifest)
+
+    await validateEdgeFunctionsManifest(manifest, featureFlags)
+    return { metrics }
   } catch (error) {
     tagBundlingError(error)
 
     throw error
   }
-
-  await validateEdgeFunctionsManifest({ buildDir, constants: { EDGE_FUNCTIONS_DIST: distDirectory }, featureFlags })
-
-  return {}
 }
 
+const getMetrics = (manifest): Metrics => {
+  const numGenEfs = Object.values(manifest.function_config).filter(
+    (config: { generator?: string }) => config.generator,
+  ).length
+  const totalEfs = manifest.routes.length + manifest.post_cache_routes.length
```

Comment:
doubt: if those are paths that can be executed by the same function, is that total number right? 🤔

Reply:
I think this is a good point! I could try to check for names!